### DISCUSSION TITLE
Unable to build web module

### DISCUSSION THREAD

**Discussion by JmzTaylor** [rustdesk/rustdesk#837] [Q&A]
I am getting this error when running yarn build.  Following directions [here](https://rustdesk.com/docs/en/dev/build/web/).

src/common.ts:2:58 - error TS2307: Cannot find module './message' or its corresponding type declarations.

2 import { KeyEvent, controlKeyFromJSON, ControlKey } from "./message";
                                                           ~~~~~~~~~~~

src/connection.ts:2:26 - error TS2307: Cannot find module './message.js' or its corresponding type declarations.

2 import * as message from "./message.js";
                           ~~~~~~~~~~~~~~

src/connection.ts:433:31 - error TS7006: Parameter 'f' implicitly has an 'any' type.

433       vf.vp9s.frames.forEach((f) => {
                                  ~

src/gen_js_from_hbb.ts:1978:5 - error TS1117: An object literal cannot have multiple properties with the same name.

1978     "Account": "帳戶",
         ~~~~~~~~~

src/websock.ts:1:26 - error TS2307: Cannot find module './message.js' or its corresponding type declarations.

1 import * as message from "./message.js";
                           ~~~~~~~~~~~~~~

Found 5 errors in 4 files.

Errors  Files
     1  src/common.ts:2
     2  src/connection.ts:2
     1  src/gen_js_from_hbb.ts:1978
     1  src/websock.ts:1
error Command failed with exit code 2.

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/discussions/763

**Comment by raminfp** [comment]
Please upgrade your `node/npm` version: 
```
For example : 

# node -v
v16.20.1

# npm -v
9.8.1


```
